metaseq/dataclass/configs.py

```diff
@@ -498,6 +498,24 @@ class CheckpointConfig(MetaseqDataclass):
         default=True,
         metadata={"help": "store a last checkpoint at the end of the training run."},
     )
+    eval_command: Optional[str] = field(
+        default=None,
+        metadata={
+            "help": (
+                "generic command to kick off evaluations. Command requirements: "
+                "1/ Wait until all checkpoints are uploaded"
+                "2/ Merge args --model-cloud-path and --model-cloud-filename to "
+                "fetch the checkpoint files"
+            )
+        },
+    )
+    evaluate_interval_updates: int = field(
+        default=0, metadata={"help": "run eval_command every N updates"}
+    )
+    evaluate_last_checkpoint: bool = field(
+        default=False,
+        metadata={"help": "run the eval_command at the end of the training run"},
+    )
```

Comment:
should we just tie this to validation?

Reply:
Sure, but I believe we are reaching 20+ tasks now. $$$$

Reply:
Okay Sgtm but Option creep is historically one of the most frustrating things, so I look to always say “do we need that flag?”